frontend/src/scenes/session-recordings/templates/sessionRecordingTemplatesLogic.tsx

```diff
@@ -0,0 +1,142 @@
+import { actions, events, kea, key, listeners, path, props, reducers, selectors } from 'kea'
+import { router } from 'kea-router'
+import { urls } from 'scenes/urls'
+
+import {
+    FilterLogicalOperator,
+    PropertyFilterType,
+    PropertyOperator,
+    RecordingUniversalFilters,
+    ReplayTabs,
+    ReplayTemplateCategory,
+    ReplayTemplateType,
+    ReplayTemplateVariableType,
+    UniversalFiltersGroupValue,
+} from '~/types'
+
+import type { sessionReplayTemplatesLogicType } from './sessionRecordingTemplatesLogicType'
+
+const getPageviewFilterValue = (pageview: string): UniversalFiltersGroupValue => {
+    return {
+        key: 'visited_page',
+        value: pageview,
+        operator: PropertyOperator.IContains,
+        type: PropertyFilterType.Recording,
+    }
+}
+
+const getFlagFilterValue = (flag: string): UniversalFiltersGroupValue => {
+    return {
+        id: '$feature_flag_called',
+        name: '$feature_flag_called',
+        type: 'events',
+        properties: [
+            {
+                key: `$feature/${flag}`,
+                type: PropertyFilterType.Event,
+                value: ['false'],
+                operator: PropertyOperator.IsNot,
+            },
+            {
+                key: `$feature/${flag}`,
+                type: PropertyFilterType.Event,
+                value: 'is_set',
+                operator: PropertyOperator.IsSet,
+            },
+            {
+                key: '$feature_flag',
+                type: PropertyFilterType.Event,
+                value: flag,
+                operator: PropertyOperator.Exact,
+            },
+        ],
+    }
+}
+
+export interface ReplayTemplateLogicPropsType {
+    template: ReplayTemplateType
+    // one card can be in multiple categories,
+    // key on the category so that multiple instances of the same card are isolated
+    category: ReplayTemplateCategory
+}
+
+export const sessionReplayTemplatesLogic = kea<sessionReplayTemplatesLogicType>([
+    path(() => ['scenes', 'session-recordings', 'templates', 'sessionReplayTemplatesLogic']),
+    props({} as ReplayTemplateLogicPropsType),
+    key((props) => `${props.category}-${props.template.key}`),
```

Comment:
![2024-10-22 09 26 42](https://github.com/user-attachments/assets/4015e6b8-35f6-4b65-941d-51c9837571b2)

duplicate cards would share interactions so let's key on the current category as well as the template

Reply:
oh yeah I meant to do that, thanks!